cobbler/actions/reposync.py

```diff
@@ -267,12 +275,8 @@ def createrepo_walker(self, repo, dirname: str, fnames):
             blended = utils.blender(self.api, False, repo)
             flags = blended.get("createrepo_flags", "(ERROR: FLAGS)")
             try:
-                cmd = "createrepo %s %s %s" % (
-                    " ".join(mdoptions),
-                    flags,
-                    pipes.quote(dirname),
-                )
-                utils.subprocess_call(cmd)
+                cmd = ["createrepo"] + mdoptions + [flags, pipes.quote(dirname)]
```

Comment:
`mdoptions` composed the way like this `mdoptions.append(f"-g {os.path.join(origin_path, groupmdfile)}")`, so it looks like the command will be executed with the single argument `-g /path/to/the/file` instead of passing `-g` and `/path/to/the/file` as distinct arguments. Is it correct here?

Reply:
Check for the rest part of the changes. It seems there are few more similar cases.

Reply:
I only found usages in reposync. Could you please check also? I don't want to miss anything.

Reply:
Looks this test fail is also related to the same case:
```
E       Expected: subprocess_call(['/my/fake/reposync', '--testflag', '--repo=testrepo0', '-p /srv/www/cobbler/repo_mirror'], shell=False)
E       Actual: subprocess_call(['/my/fake/reposync', '--testflag', '--repo=testrepo0', '-p=/srv/www/cobbler/repo_mirror'], shell=False)
```